lab_1/main.py

```diff
@@ -12,8 +13,16 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
-
+    if type(text) == str:
```

Comment:
1. is linting with `type()` check passes? if not, write `isinstance(text, str)` here
2. better write this condition as `if not type(text) == str: return []` and then implement main logic of the function without any nesting